### DISCUSSION TITLE
Config issues with XT8

### DISCUSSION THREAD

**Discussion by w0lf3h** [Vaskivskyi/ha-asusrouter#166] [Q&A] [answered]
Hey there :)

Trying to get this integration setup with my XT8. First off, the config flow doesn't allow me to enter the SSL certificate path till after going through the config flow at least once. Then I have to go back to "configure" the integration to add it.

Then, when I do try adding the cert file, I get this error in the config flow.

![Screenshot 2022-07-27 122651](https://user-images.githubusercontent.com/1549104/181355701-a9006147-8695-4e21-a2b4-6448067e49ec.png)

I think I'm doing something wrong 😅 I assume it's probably path related, but I can't figure out how to fix it. I am supposed to be using the certificate exported from my admin panel within the router, right? 

Thanks 😄

**Comment by Vaskivskyi** [comment, marked as answer]
With the latest versions of AsusRouter, the SSL certificate feature is deprecated and removed. Because of it, I am closing this discussion the same way the https://github.com/Vaskivskyi/ha-asusrouter/issues/171 was closed
